core/src/modules/cache.ts

```diff
@@ -67,7 +70,7 @@ function makeCacheString(cacheKey: CacheKey) {
       const date: Date = <Date>cacheKey;
       return date.toISOString();
     } else if (type === "Array") {
-      const array = <Array<CacheKey>>cacheKey;
+      const array = <CacheKey[]>cacheKey;
```

Comment:
```suggestion
      const array: CacheKey[] = cacheKey;
```
A linter should ideally flag this.

Reply:
Whoops! this was on purpose - `cacheKey<t>` accepts a type